x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/action.ts

```diff
@@ -36,6 +37,16 @@ interface ServerFailedToReturnEndpointDetails {
   payload: ServerApiError;
 }
 
+interface ServerReturnedEndpointDetailsActivityLog {
```

Comment:
Instead of using this older style store actions for async requests, look to maybe used `AsyncResourceState` instead

Reply:
Indeed. It's on my list of things to do. I recall we discussed this couple of days ago.

Reply:
done c5ade7bd293be1f5ef9e19873c68513f16f399cf